### ISSUE TITLE
Serve a form's media files with the form

### ISSUE THREAD

**Issue by abbyad** [medic/medic-webapp#979]
Following #886,  we need to be able to serve the media files necessary to render the blank form. On the phone these are kept in a folder named `{{form_name}}-media`. For example, form `registration` would have all associated media in the `registration-media` folder. 

The most common of these files are the image shown on the first page of the form (`form_logo.png`) and the data to populate fields or lists (`itemsets.csv`). Additionally, any audio, image, or video that is displayed as part of a form question would be kept in this folder and needs to be served.

The API for media content is described [here](https://bitbucket.org/javarosa/javarosa/wiki/FormListAPI) as "The Manifest Document".

**Comment by mandric** [comment]
Can you provide a working example of a form with a manifest file?

**Comment by abbyad** [comment]
The manifest file is only used when serving forms, not in manually putting the files on the device. The manifest file can be generated by [openrosa-manifest](https://github.com/digidem/openrosa-manifest), and you can see an example [here](https://github.com/digidem/openrosa-manifest/blob/master/index.js)

**Comment by mandric** [comment]
That example refers to the wrong library oddly enough.  If you have any specific examples in mind you want to see working that I can use to prove my implementation please include them, thanks!

**Comment by abbyad** [comment]
No specific examples... you can just use a `form_logo.png` file, which by default will be shown when you open the form on the device so will be easy to know that it worked.

**Comment by mandric** [comment]
Can we close this?

**Comment by abbyad** [comment]
I don't think we should. Any Collect user updating their forms would not get any media/data files attached to their forms.

**Comment by abbyad** [comment]
Assigning to @sglangevin to schedule as needed.

**Comment by sglangevin** [comment]
@abbyad is this needed for any particular project? I'm trying to figure out whether this is an urgent priority.

**Comment by abbyad** [comment]
This feature request is needed to properly support Collect, but I am not aware of this particular feature being needed for a project. That is likely because our team doesn't know about it rather than it not being useful. Perhaps we can get @ngamita to comment on usefulness for current/planned projects with Collect, and schedule accordingly.

**Comment by ngamita** [comment]
We needed this feature for IsraAid at some point where they wanted to have an image of the "MUAC band"  on collect and its different colors. MUAC band is used to measure the circumference of a child’s upper arm. If the measurement is green, the child is normal. If the measurement is yellow, the child has moderate malnutrition. And if the measurement is red, the child has acute malnutrition. It would have been easier to have the image on collect for CHWs to reference but we didn't use it. I foresee more projects requiring this in the future but for now this is not an URGENT priority.
